src/EntityFramework.SqlCe.Legacy.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>EntityFramework.SqlCe.Legacy</id>
```

Comment:
I tried to keep the same naming scheme as was used for the EF6 packages, while I await your final package naming decision! Ideally, I would like EntityFramework.SqlServerCompact and EntityFramework.SqlServerCompact.Legacy, but they were taken by someone called "Microsoft"